ctgan/synthesizers/tvae.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         decompress_dims=(128, 128),
         l2scale=1e-5,
         batch_size=500,
+        verbose=False,
```

Comment:
can we add this parameter to the end? Just in case anybody is passing in all the parameters unnamed, we don't want the wrong values being passed to the wrong parameters

Reply:
Can be passed to the end, although this ordering is consistent with CTGAN class (https://github.com/sdv-dev/CTGAN/blob/2848a42651c440a01a987a27658722069c7bb9b1/ctgan/synthesizers/ctgan.py#L145). Would it be better to keep the two orderings consistent?

Reply:
Hmm I guess it's fine to keep consistent.